src/D2L.Security.OAuth2/JsonSerializer.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace D2L.Security.OAuth2 {
+	internal static class JsonSerializer {

```

Comment:
Consolidating on the System.Text.Json-looking API, hiding some ugly `#if`s in this file (unfortunately I can't hide them all)